accessories/he_st_accessories.js

```diff
@@ -974,6 +974,7 @@ function convertAlarmState(value, valInt = false) {
         case 'armnight':
         case 'armNight':
         case 'armednight':
+        case 'armedNight':
```

Comment:
@spectre3ooo Can you confirm this change fixes the issue?
Off the top of my head I think that groovy code will need to be changed as well.

Reply:
It seems to be working fine for me with just this change.  I had to go in and add sensors to Home and Away before they would arm, but now I can cycle through them all.

Reply:
Maybe I doesn’t need a groovy change....  Thanks for submitting.  I’m merging now